spec/features/users/user_profile_spec.rb

```diff
@@ -19,13 +19,12 @@
   end
 
   context 'user clicks the delete account button' do
-    xit 'deletes the account and signs the user out with a flash message' do
-      pending 'temporarily disabled until we figure out the MBUN to SSN mapping'
```

Comment:
This reminds, what happens if I create an LOA3 account, then delete my account, and then try to create another LOA3 account with the same SSN? It should work since the deletion should cascade through the profile, but it may be worth covering with a regression test. Thoughts?

Reply:
I agree.  I validated delete account does not delete identities so it make sense to validate the deletes that should cascade actually do specifically the ones with PII like profile.